submit_for_testing.py

```diff
@@ -120,15 +120,16 @@ def _submit_to_squad(lava_job, lava_url_base, qa_server_api, qa_server_base, qa_
             ).netloc,  # qa-reports backends are named as lava instances
         }
         logger.info("Submit to: %s" % qa_server_api)
-        results = requests.post(qa_server_api, data=data, headers=headers, timeout=31)
-        if results.status_code < 300:
-            logger.info(
-                "%s/testjob/%s %s"
-                % (qa_server_base, results.text, get_job_name(lava_job))
+        try:
+            results = requests.post(
```

Comment:
The variable `results` should probably be named `response` instead since that is what is being returned.

https://requests.readthedocs.io/en/latest/api/#requests.post